src/app/main.cpp

```diff
@@ -425,3 +444,34 @@ void adjustFileDescriptorLimit()
     setrlimit(RLIMIT_NOFILE, &limit);
 }
 #endif
+
+void handleChangedDefaults(const DefaultPreferencesMode mode)
+{
+    using DefaultValue = struct
```

Comment:
I don't insist on this but why not use the old & shorter `struct DefaultValue {/*...*/};` declaration?

Reply:
:+1:
For some reason, I was sure that I could not declare classes inside functions in the usual way... did I mix up the programming languages? Or did I just forget something?...

Reply:
PR updated.